core/domain/exp_services_test.py

```diff
@@ -2073,7 +2073,7 @@ def test_export_to_zip_file_with_assets(self):
             zf.namelist(), ['A title.yaml', 'assets/image/abc.png'])
         self.assertEqual(
             zf.open('A title.yaml').read(), self.SAMPLE_YAML_CONTENT)
-        self.assertEqual(zf.open('assets/image/abc.png').read(), raw_image)
+        self.assertEqual(zf.open('assets/image/abc.png').read(), raw_image) # pylint: disable=disallowed-function-calls
```

Comment:
Why do we need this change? (Ditto below)

Reply:
Cleaned up the useless pragmas.